docs/src/tutorials/object_detection.md

```diff
@@ -0,0 +1,122 @@
+representation which is invariant to local geometric and photometric changes (i.e. shape and illumination changes) and so is a good choice for our problem. In fact
+HOG descriptors are widely used for object detection.
+
+We will use [this data](https://drive.google.com/file/d/0B9V0KF3ZHWtWa0hGNHJDSm1XZnM/view?usp=sharing) for training our classifier. We will start by loading the data and computing HOG features of all the images.
+
+```julia
+using Images, ImageFeatures
+
+pos_examples = "path_to_data/human/"
+neg_examples = "path_to_data/not_humans/"
+
+n_pos = length(readdir(pos_examples))   # number of positive training examples
+n_neg = length(readdir(neg_examples))   # number of negative training examples
+n = n_pos + n_neg                       # number of training examples 
+data = Array{Float64}(n, 3780)          # Array to store HOG descriptor of each image. Each image in our training data has size 128x64 and so has a 3780 length 
+labels = Vector{Int}(n)                 # Vector to store label (1=human, 0=not human) of each image.
+
+i = 0
+for file in [readdir(pos_examples); readdir(neg_examples)]
+    i += 1
+    filename = "$(i <= n_pos ? pos_examples : neg_examples )$file"
+    img = load(filename)
+    examples[i, :] = create_descriptor(img, HOG())
+    labels[i] = (i <= n_pos ? 1 : 0)
+end
+```
+
+Basically we now have an encoded version of images in our training data. This encoding captures useful information but discards extraneous information 
+(illumination changes, pose variations etc). We will train a linear SVM on this data.
+
+```julia
+using LIBSVM
+
+#Split the dataset into train and test set. Train set = 2500 images, Test set = 122 images.
+random_perm = randperm(n)
+train_ind = random_perm[1:2500]
+test_ind = random_perm[2501:end]
+
+model = svmtrain(data[:, train_ind], labels[train_ind]);
+```
+
+Now let's test this classifier on some images.
+
+```julia
+img = load("humans/per00003.ppm")
+descriptor = Array{Float64}(3780, 1)
+descriptor[:, 1] = create_descriptor(img, HOG())
+
+predicted_label, _ = svmpredict(model, descriptor);
+print(predicted_label)                          # 1=human, 0=not human
+
+# Get test accuracy of our model
+(predicted_labels, decision_values) = svmpredict(model, data[:, test_ind]);
+@printf "Accuracy: %.2f%%\n" mean((predicted_labels .== labels[test_ind]))*100 # test accuracy should be > 98%
+```
+
+Try testing our trained model on more images. You can see that it performs quite well.
+
+| ![Original](../img/human1.png) | ![Original](../img/human2.png) |
+|:------:|:---:|
+| predicted_label = 1 | predicted_label = 1 |
+
+| ![Original](../img/human3.png) | ![Original](../img/not-human1.jpg) |
+|:------:|:---:|
+| predicted_label = 1 | predicted_label = 0 |
+
+Next we will use our trained classifier with a sliding window to localize persons in an image.
+
+![Original](../img/humans.jpg)
+
+```julia
+img = load("humans.jpg")
```

Comment:
Maybe `load("$pos_examples/../humans.jpg")`? Or tell people to navigate to the data directory and get rid of your `path_to_data` above.